sdk/src/utils/pdf_utils.rs

```diff
@@ -193,14 +193,14 @@ impl C2paPdf for Pdf {
     /// This method will read the bytes of the manifest, whether the manifest was added to the
     /// PDF via an `Annotation` or an `EmbeddedFile`.
     ///
-    /// Returns an [Ok(None)] if no manifest is present. Returns a [Ok(Some(&Vec<u8>))] when a manifest
+    /// Returns an `Ok(None)` if no manifest is present. Returns a `Ok(Some(Vec<&[u8]>))` when a manifest
     /// is present.
-    fn read_manifest_bytes(&self) -> Result<Option<&Vec<u8>>, Error> {
+    fn read_manifest_bytes(&self) -> Result<Option<Vec<&[u8]>>, Error> {
         if !self.has_c2pa_manifest() {
             return Ok(None);
-        }
+        };
 
-        Ok(Some(
+        Ok(Some(vec![
```

Comment:
I don't understand why this API returns a vector here. Is there ever any case where there would be more than one byte slice that could be returned?

Reply:
I'll update the docs to reference the spec with a quick explanation, if you are okay with the `Vec<&[u8]>` return type

Reply:
OK, that makes sense. Please do update the docs.